opentelemetry-auto-instrumentation/README.rst

```diff
@@ -0,0 +1,19 @@
+OpenTelemetry Auto Instrumentation
```

Comment:
This should have a more elaborate README, e.g. on using the console scripts or setting PYTHONPATH manually.